### ISSUE TITLE
text encoder

### ISSUE THREAD

**Issue by dt-yuhui** [cheliu-computation/MERL-ICML2024#3]
I noticed that the function get_text_emb is decorated with @torch.no_grad(), which means the text encoder's parameters won't be updated. However, I also noticed that in config.yaml it says free_layers: 6 # set 12 to freeze all layers in bert, so is your text encoder frozen?

**Comment by dt-yuhui** [comment]
Additionally, there are many parameters in your config.yaml that differ from those in the paper. For example, the learning rate and weight decay in the paper are 2e-4 and 1e-5 respectively, but in the config file, they are 1e-3 and 1e-8.

**Comment by cheliu-computation** [comment]
Thanks for your detailed response! There should be typo in the config file, please change it to match the parameters mentioned in paper.
